hera_pspec/pspecbeam.py

```diff
@@ -491,17 +493,16 @@ def __init__(self, OmegaP, OmegaPP, beam_freqs, cosmo=None):
             Cosmology object. Uses the default cosmology object if not 
             specified. Default: None.
         """
-        self.allowed_pols = ['pI', 'pQ', 'pU', 'pV', 
-                             'XX', 'YY', 'XY', 'YX']
         self.OmegaP = {}; self.OmegaPP = {}
+        self.allowed_pols = [1, 2, 3, 4, -5, -6, -7, -8]
```

Comment:
These magic numbers need to be documented.

Reply:
Not sure what you mean. They are, they are the AIPS convention for polarization integers. See `pyuvdata.utils.polstr2num` and `pyuvdata.utils.polnum2str` for how these are made. I can make a comment pointing to them in the code, though